x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authz/store/NativeRolesStoreTests.java

```diff
@@ -78,18 +82,19 @@ public void terminateThreadPool() throws Exception {
         terminate(threadPool);
     }
 
-    // test that we can read a role where field permissions are stored in 2.x format (fields:...)
-    public void testBWCFieldPermissions() throws IOException {
+    // test that we reject a role where field permissions are stored in 2.x format (fields:...)
+    public void testBWCFieldPermissions() throws Exception {
+        Logger logger = CapturingLogger.newCapturingLogger(Level.INFO, null);
+        List<String> events = CapturingLogger.output(logger.getName(), Level.ERROR);
+        events.clear();
         Path path = getDataPath("roles2xformat.json");
         byte[] bytes = Files.readAllBytes(path);
         String roleString = new String(bytes, Charset.defaultCharset());
-        RoleDescriptor role = NativeRolesStore.transformRole(RoleDescriptor.ROLE_TYPE + "role1",
-            new BytesArray(roleString), logger, new XPackLicenseState(Settings.EMPTY));
-        assertNotNull(role);
-        assertNotNull(role.getIndicesPrivileges());
-        RoleDescriptor.IndicesPrivileges indicesPrivileges = role.getIndicesPrivileges()[0];
-        assertThat(indicesPrivileges.getGrantedFields(), arrayContaining("foo", "boo"));
-        assertNull(indicesPrivileges.getDeniedFields());
+        assertNull(NativeRolesStore.transformRole(RoleDescriptor.ROLE_TYPE + "_role1",
```

Comment:
note that I added the underscore here - did that based on the transformRole implementation, rather than expecting role "ole1"